doc/userdoc/contribute/templates_styleguides/styleguide.rst

```diff
@@ -0,0 +1,623 @@
+  way
+- factual: we write in a neutral style avoiding irony, humor, or cultural references
+
+Use 'you' to indicate the reader and 'we' to indicate yourself (the writer and
+possibly the NEST team).
+
+The language we use
+~~~~~~~~~~~~~~~~~~~
+
+We follow spelling and grammar rules of American English.
+
+The markup language we use is reStructuredText. This includes documentation
+blocks in files written in Python and cpp.
+
+We avoid slang or jargon that is unnecessary or could be confusing to readers.
+
+We do, however, use terminology appropriate to NEST and neuroscience when
+specific and exact terms are needed, which is often. But explanations or
+references should be provided for clarity in introductory texts and the glossary.
+
+If you have further questions about style not addressed here, see <<STANDARDIZED STYLE GUIDE>>
+or ask on the mailing list.
+
+Active voice
+~~~~~~~~~~~~
+
+Prefer the active voice, where the subject acts on object.
+
+If the object needs emphasis rather than subject, use passive voice. But ensure
+that you cannot improve the sentence by using the active voice.
+
+
+reStructuredText markup and formatting
+--------------------------------------
+
+reStructuredText is a plain text markup language and parser. It is the default language of the Sphinx documentaiton
+generator, which NEST uses for generating documentation.
+
+reStructuredText uses directives, which are blocks of explicit markup used for math, images, code, admonitions and much
+more. The syntax looks like this ``.. directive-name::``. The directive content follows after a blank line and is indented
+relative to the directive start.
+
+In addition to directives, reStructuredText has roles, which insert semantic markup into documents.
+Roles look like this ``:role-name:`content```.
+
+We will only cover a few examples here. You can find more information in the following links.
+
+
+* `reStructuredText User Documentation <https://docutils.sourceforge.io/rst.html#id24>`_
+
+* `reStructuredText Primer <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_
+
+* `Sphinx directives <https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html>`_
+
+* `Sphinx roles <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`_
+
+
+.. note::
+
+   reStructuredText is sensitive to indentation and new lines.
+
+   * Directives, headings, labels, and tables should be separated from other text by a new line, excluding directive options.
+   * Directive options must be indented to the same level as the directive content.
+   * Text in multiline content should stay aligned with the first line.
+   * If the output format seems incorrect, double check the spaces and newlines
+     of the text.
+
+NumPy style docstrings
+~~~~~~~~~~~~~~~~~~~~~~
+
+In PyNEST code, we follow the rules for NumPy style docstrings as
+`explained here <https://numpydoc.readthedocs.io/en/latest/format.html>`_.
```

Comment:
Should we mention explicitly here that this implies that in PyNEST documentation, one shall only use the sections listed in the NumPy doc guide, in that order, and not use any admonitions? Or at least a sentence explicitly asking documenters to carefully read the NumPy Guide before working on PyNEST docstrings?

Reply:
Added text